cloudsync/smartsync.py

```diff
@@ -0,0 +1,191 @@
+from typing import Optional, Tuple, TYPE_CHECKING, Callable, List
+from cloudsync import CloudSync, SyncManager, SyncState, SyncEntry
+from cloudsync.types import LOCAL, REMOTE, DIRECTORY, DirInfo
+
+if TYPE_CHECKING:
+    from .provider import Provider
+    from cloudsync import Storage
+    
+
+class SmartSyncManager(SyncManager):
```

Comment:
Please add smartsync modules to `cloudsync.__init__.py`

Reply:
done